src/transmuxer_worker.js

```diff
@@ -74,6 +74,9 @@ var messageHandlers = {
    * default options if `init` was never explicitly called
    */
   defaultInit: function () {
+    if (transmuxer) {
+      transmuxer.dispose();
```

Comment:
This is just a fix that I decided to sneak in here. It wasn't breaking anything but I didn't like the idea of there being the possibility that the *old* transmuxer (if there was one) was still emitting events that we could be pumping through the web worker interface.